### PR TITLE
CAMEL-13025:FileDeletionWithZeroTimeout

### PR DESCRIPTION
**Pull request by vgvineet4**

### CODE REVIEW

File: camel-core/src/main/java/org/apache/camel/component/file/strategy/FileChangedExclusiveReadLockStrategy.java [hunk 1]
```diff
@@ -71,6 +71,11 @@ public boolean acquireExclusiveReadLock(GenericFileOperations<File> operations,
                 }
             }
 
+            if(!target.exists()) {
```
[the remark this answers is not in the document]
Comment (davsclaus) [reply camel-core/src/main/java/org/apache/camel/component/file/strategy/FileChangedExclusiveReadLockStrategy.java]: Maybe reword it as `the file no longer exists.`

**Comment by davsclaus** [comment]
Okay I got a bit time to polish this PR.

@vgvineet4 thanks for your first PR to Apache Camel. You are welcome to keep contributing.

**Comment by onderson** [comment]
added a unit test. fyi.
